### PR TITLE
Wizard 364

### PR DESCRIPTION
**Pull request by taynaps**
#364 adding the wizard

### CODE REVIEW

File: src/main/java/de/qabel/desktop/ui/accounting/wizard/WizardController.java [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+            loadCurrentStep();
+        }
+    }
+
+    public void back() {
+        if (currentStepIndex.get() > 0) {
+            loadStep.getChildren().get(currentStepIndex.get()).setVisible(false);
+            currentStepIndex.set(currentStepIndex.get() - 1);
+            currentStep = steps.get(currentStepIndex.get());
+            loadCurrentStep();
```
Comment (audax) [inline src/main/java/de/qabel/desktop/ui/accounting/wizard/WizardController.java]: Can't you just bind loadCurrentStep on any change of the currentStepIndex?

Comment (taynaps) [inline src/main/java/de/qabel/desktop/ui/accounting/wizard/WizardController.java]: changed!

File: src/test-gui/java/de/qabel/desktop/ui/sync/SyncControllerGuiTest.java [hunk 2]
```diff
@@ -44,7 +44,6 @@ public void syncSetupIntegration() {
         setup.enterRemotePath("/");
         setup.startSync();
 
-        waitUntil(() -> !controller.addStage.isShowing());
```
Comment (audax) [inline src/test-gui/java/de/qabel/desktop/ui/sync/SyncControllerGuiTest.java]: Why is this test related to your PR?

File: src/main/resources/ui.properties [hunk 3]
```diff
@@ -218,3 +218,14 @@ phoneLabel=Phone
 aliasLabel=Alias
 indexSearchPrompt=Enter a phone number or email address
 assignContactTitle=assign to identities
+next=NEXT
+back=BACK
+finish=FINISH
+nameIdentityInstructions=Please enter the name of the identity, e.g.: "Jane" or "Forentroll"
+nameIdentityPromptText=Name or pseudonym of identity
+emailAddress=E-Mail Address
+emailInstructions=Enter an email address to be found by other Qabel users
```
Comment (julianseeger) [inline src/main/resources/ui.properties]: in the Qabel Index Server
(same applies for the phoneNumber which instructions are too short here)

File: src/main/java/de/qabel/desktop/ui/accounting/wizard/WizardController.java (hunk 1 above)
Comment (julianseeger) [inline src/main/java/de/qabel/desktop/ui/accounting/wizard/WizardController.java]: ✓

File: src/test-gui/java/de/qabel/desktop/ui/sync/SyncControllerGuiTest.java (hunk 2 above)
Comment (julianseeger) [inline src/test-gui/java/de/qabel/desktop/ui/sync/SyncControllerGuiTest.java]: ✓ flaky

File: build.gradle [hunk 4]
```diff
@@ -127,6 +127,7 @@ dependencies {
     compile 'net.glxn.qrgen:javase:2.+'                     //Apache License https://github.com/kenglxn/QRGen
     compile 'org.controlsfx:controlsfx:8.+'
     compile 'com.jfoenix:jfoenix:1.+'
+    compile group: 'commons-validator', name: 'commons-validator', version: '1.4.0'
```
Comment (julianseeger) [inline build.gradle]: ✓ still not the short version but works anyways
